info/as_parser.go

```
@@ -418,7 +410,19 @@ func ParseNamespaceNames(m map[string]string) []string {
 
 // ParseDCNames parses all DC names
 func ParseDCNames(m map[string]string) []string {
-	return getNames(m[constStatDCNames])
+	rawXDRConfig, exists := m[cmdConfigXDR]
+	if !exists || rawXDRConfig == "" {
+		return []string{}
+	}
+
+	xdrConfig := ParseIntoMap(rawXDRConfig, ";", "=")
+
+	rawNames, ok := xdrConfig[constStatDCNames].(string)
+	if !ok || rawNames == "" {
```

if not ok , which means parsing failure. should we log this ... else error will be silently dropped with empty string

---

Resolved after discussion, as we are not logging parsing problems under the current scope.